aws-event-sources-operator/deploy/olm-catalog/aws-event-sources-operator/manifests/sources.triggermesh.com_awseventsources_crd.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: awseventsources.sources.triggermesh.com
+spec:
+  group: sources.triggermesh.com
+  names:
+    kind: AwsEventSources
+    listKind: AwsEventSourcesList
+    plural: awseventsources
+    singular: awseventsources
+  scope: Namespaced
+  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        type: object
+        x-kubernetes-preserve-unknown-fields: true
+    served: true
+    storage: true
+    subresources:
+      status: {}
```

Comment:
We need 2 copies of the exact same CRD file?

Reply:
yes. they are the exact same CRD from the deploy/ folder.

The `deploy` folder contains the development version of the CRD. at the time of release the `bundle/X.Y.Z` folder is created and the manifests from the deploy folder are copied over and the package.yaml file is updated. The bundle folder should then be zipped and sent to redhat for updation in the catalog.

The files generated and the duplicates exists out of the process laid out by redhat (https://redhat-connect.gitbook.io/certified-operator-guide/helm-operators/building-a-helm-operator).

Reply:
Got it, could you please document the procedure in the operators' README or the DEVELOPMENT.md file so that we have something to refer to? If the procedure is documented somewhere else, basic steps with links would suffice.

Reply:
that makes sense. will add some info